packages/flutter_tools/lib/src/commands/packages.dart

```diff
@@ -124,6 +132,36 @@ class PackagesTestCommand extends FlutterCommand {
   }
 }
 
+class PackagesForwardCommand extends FlutterCommand {
+  PackagesForwardCommand(this._commandName, this._description, {bool requiresPubspec = false}) {
+    if (requiresPubspec) 
```

Comment:
It looks like the if statement here has no body.

```suggestion
    if (requiresPubspec) {
      requiresPubspecYaml();
    }
```

Reply:
Fixed